### DISCUSSION TITLE
are linux builds of carla behind schedule? latest linux build is 2.2.0 but latest win/mac is 2.5.9

### DISCUSSION THREAD

**Discussion by joshgura** [falkTX/Carla#1936] [General]
there's no carla release at the github page that i could find, but the old version 2.2.0 is at the main page. is there a 2.5.9 build for linux or do we build our own?

**Comment by falkTX** [comment]
you can grab the deb package from https://kx.studio/Repositories:Applications which is updated

**Reply by joshgura** [reply]
thanks for the reply! i'm on arch, sometimes i can just extract a deb and find the binary and run it, but that's not the case with this particular package. gave everything that looked like a binary permissions and tried to execute and they closed instantly. (in terminal, the error message was due to the app needing to be installed at /usr/bin ... or something)

anyways, i try to use standalone apps whenever possible but in this case i see there's a version 2.5.8-3 on pamac so i will give that a go.

**Reply by falkTX** [reply]
then just install it from the arch repos?

there is no need for that old binary
